ansible/roles/debops.postldap/COPYRIGHT

```diff
@@ -0,0 +1,18 @@
+debops.postldap - Configure Postfix SMTP service to use Virtual Mail with LDAP support
+
+Copyright (C) 2019 Rainer 'rei' Schuth <devel@reixd.net>
+Copyright (C) 2014-2017 DebOps https://debops.org/
```

Comment:
You should change the year to `2019` since that's when this role has been introduced to the project.